lib/screens/v2/widget_instance/dup_alert.ex

```diff
@@ -139,26 +140,26 @@ defmodule Screens.V2.WidgetInstance.DupAlert do
   # Table is adapted from the one in the product spec: https://www.notion.so/mbta-downtown-crossing/DUP-Alert-Widget-Specification-a82acff850ed4f2eb98a04e5f3e0fe52
   # Compile-time code converts each table row to `do_alert_layout` function clauses.
```

Comment:
Maybe add a lil comment noting that TotalSections refers to sections in primary_departures (so folks don't think it's total number of sections for both primary & secondary)